### PR TITLE
[Docs #46] add megamenu

### PR DESCRIPTION
**Pull request by samrichca**
Ticket: overture-stack/roadmap#46

Description:

- Convert Products menu to a reusable megamenu
- CSS changes (add borders/shadows to megamenu, reduce padding/margins in nav to accommodate new items)
- Add a constant for showing/hiding documentation section (this will become an environment variable in the next ticket)
- Collect all colours in a SCSS file
- Changed text of the header Github button
- Changed styling of header Github & Slack buttons
- Add Documentation to menus
- Update readme

## requires SHOW_DOCS feature flag (not merged yet)

### CODE REVIEW

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -79,89 +78,95 @@ const links = {
...
   return (
     <footer className="Footer">
       <div className="container">
         <div className="footer-box">
           <a className="oicr-logo" target="_blank" href="http://oicr.on.ca">
-            <img
-              src={logo}
-              style={{ maxHeight: '100px', paddingRight: '32px' }}
-            />
+            <img src={logo} />
```
Comment (anncatton) [inline src/components/Footer/index.js:161]: add an `alt` attr

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -172,11 +177,11 @@ const Footer = () => {
       </div>
       <div className="container netlify-badge">
         <a target="_blank" href="https://www.netlify.com">
-          <img src="https://www.netlify.com/img/global/badges/netlify-color-bg.svg"/>
+          <img src="https://www.netlify.com/img/global/badges/netlify-color-bg.svg" />
```
Comment (anncatton) [inline src/components/Footer/index.js:180]: `alt` attr

File: src/constants.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export const SHOW_DOCS = true;
```
Comment (anncatton) [inline src/constants.js:1]: is this going to come from an env/config at some point?

File: src/pages/privacy/styles.scss [hunk 4]
```diff
@@ -30,8 +34,9 @@
     counter-increment: customlistcounter;
   }
 
-  ol > li:before { /* the only way to remove "." from an OL ?! */
-    content: counter(customlistcounter) " ";
+  ol > li:before {
+    /* the only way to remove "." from an OL ?! */
```
Comment (anncatton) [inline src/pages/privacy/styles.scss:38]: i think this can just be `list-style: none` ?

File: src/styles/_colors.scss [hunk 5]
```diff
@@ -0,0 +1,29 @@
+$blue-bg: #6ac3e8;
```
Comment (anncatton) [inline src/styles/_colors.scss:1]: thank you for colour vars!! at some point we should standardize the names across our products :)

**Review by anncatton — COMMENTED** [review]
Just a few small comments :)

File: src/constants.js (hunk 3 above)
Comment (samrichca) [reply src/constants.js:1]: yup, it'll be environment variables - you'll see documentation in development but not production

File: src/styles/_colors.scss (hunk 5 above)
Comment (samrichca) [reply src/styles/_colors.scss:1]: agreed! I wasn't sure what to call most of these, but wanted the vars anyways.

File: src/pages/privacy/styles.scss (hunk 4 above)
Comment (samrichca) [reply src/pages/privacy/styles.scss:38]: nah, it's an ordered list, and the original developer wanted to change `1.` to `1` (no period)

**Review by anncatton — APPROVED** [review]
Merge once the `SHOW_DOCS` feature flag is set up!

**Pull request merged**